src/libs/OptionsListUtils.js

```diff
@@ -213,20 +213,22 @@ function getParticipantsOptions(participants, personalDetails) {
     const details = getPersonalDetailsForAccountIDs(_.pluck(participants, 'accountID'), personalDetails);
     return _.map(participants, (participant) => {
         const detail = details[participant.accountID];
+        const login = detail.login || participant.login;
+        const displayName = detail.displayName || LocalePhoneNumber.formatPhoneNumber(login);
         return {
             keyForList: String(detail.accountID),
-            login: detail.login,
+            login,
             accountID: detail.accountID,
-            text: detail.displayName,
+            text: displayName,
             firstName: lodashGet(detail, 'firstName', ''),
             lastName: lodashGet(detail, 'lastName', ''),
-            alternateText: Str.isSMSLogin(detail.login || '') ? LocalePhoneNumber.formatPhoneNumber(detail.login) : detail.login || detail.displayName,
+            alternateText: LocalePhoneNumber.formatPhoneNumber(login) || displayName,
```

Comment:
What is this change for? `login` is not guaranteed to be a phone number

Reply:
Just simplifying it actually 😅. LocalePhoneNumber.formatPhoneNumber(login) will return it untouched if it's not a phone number.
https://github.com/Expensify/App/blob/0b7455fa832b53eccb99ca23c70e131b3464fb1d/src/libs/LocalePhoneNumber.js#L19-L33

We also did (use LocalePhoneNumber.formatPhoneNumber(login) without extra checking) that previously here
https://github.com/bernhardoj/Expensify/commit/ef3d9558f796a6401b1a05d1ecb50c890cf120cd#diff-733528d957ae6a80eb863637fa5d54278c89c6121998f3574fa8c0aba935daebL181-L187